graphstorm-processing/graphstorm_processing/graph_loaders/dist_heterogeneous_loader.py

```diff
@@ -1029,16 +1036,26 @@ def _process_node_features(
         """
         node_type_feature_metadata = {}
         ntype_feat_sizes = {}  # type: Dict[str, int]
+
         for feat_conf in feature_configs:
-            json_representation = self.transformation_representations.get(feat_conf.feat_name, {})
+            json_representation = (
+                self.pre_computed_transformations.get("node_features", {})
+                .get(node_type, {})
+                .get(feat_conf.feat_name, {})
+            )
```

Comment:
This will get a value if there exists a pre-computed representation for this feature name and node type, None otherwise. We do the same for the edges.